### ISSUE TITLE
[Question]  grad_norm=None

### ISSUE THREAD

**Issue by Pixel-anter** [haotian-liu/LLaVA#1787]
### Question
Hello, I am currently fine-tuning the visual encoder using LLaMA2 + CLIP-14-336 with the finetune_full_schedule.sh script. However, I noticed that the loss does not decrease and fluctuates significantly. What could be the reason for this？

**Comment by Pixel-anter** [comment]
已解决
